rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -29,219 +29,437 @@
     """
-
-    kwargs_pybind = {"input_image0": inputs[0], "is_output": False, "alpha": None, "beta": None}
-    brightness_image = b.Brightness(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    # pybind call arguments
+    kwargs_pybind = {"input_image": inputs[0], "is_output": False, "alpha": alpha, "beta": beta,
+                     "rocal_tensor_output_layout": rocal_tensor_output_layout, "rocal_tensor_output_datatype": rocal_tensor_output_datatype}
+    brightness_image = b.BrightnessFixed(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return (brightness_image)
 
-def brightness_fixed(*inputs, alpha=None, beta=None, seed=-1, device=None):
-    alpha = b.CreateFloatParameter(alpha) if isinstance(alpha, float) else alpha
-    beta = b.CreateFloatParameter(beta) if isinstance(beta, float) else beta
-    kwargs_pybind = {"input_image0": inputs[0], "is_output": False, "alpha": alpha, "beta": beta}
-    brightness_image = b.Brightness(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
-    return (brightness_image)
 
-def lens_correction(*inputs, strength =None, zoom = None):
-    strength = b.CreateFloatParameter(strength) if isinstance(
-        strength, float) else strength
-    zoom = b.CreateFloatParameter(zoom) if isinstance(zoom, float) else zoom
-    kwargs_pybind = {"input_image0": inputs[0], "is_output": False, "strength": strength, "zoom": zoom}
-    len_corrected_image = b.LensCorrection(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+def lens_correction(*inputs, strength=None, zoom=None, rocal_tensor_output_layout=types.NHWC, rocal_tensor_output_datatype=types.UINT8):
+    """
+    inputs - the input image passed to the augmentation
+    
+    strength (float, optional, default = None) - strength value used for the augmentation
+
+    zoom (float, optional, default = None) - zoom value used for the augmentation
+
+    rocal_tensor_output_layout (int, optional, default = types.NHWC) - tensor layout for the augmentation output
+
+    rocal_tensor_output_datatype (int, optional, default = types.UINT8) - tensor dtype for the augmentation output
+    """
+    strength = b.createFloatParameter(strength) if isinstance(strength, float) else strength
+    zoom = b.createFloatParameter(zoom) if isinstance(zoom, float) else zoom
+
+    # pybind call arguments
+    kwargs_pybind = {"input_image": inputs[0], "is_output": False, "strength": strength, "zoom": zoom,
+                     "rocal_tensor_output_layout": rocal_tensor_output_layout, "rocal_tensor_output_datatype": rocal_tensor_output_datatype}
+    len_corrected_image = b.LensCorrection(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return (len_corrected_image)
 
-def blur(*inputs, blur=3, device=None):
+
+def blur(*inputs, kernel_size=None, rocal_tensor_output_layout=types.NHWC, rocal_tensor_output_datatype=types.UINT8):
```

Comment:
I think we need to add one liner about what this augmentation does. Let's see what Rajy says.